### PR TITLE
Variable assigned to itself in data.py

### PR DESCRIPTION
**Pull request by springcoil**
Minor update.

### CODE REVIEW

File: pymc3/data.py [hunk 1]
```diff
@@ -340,24 +351,30 @@ def check(t):
...
+                else tt.arange(shp_begin[i])
+                for i, t in enumerate(begin)
+            ]
+            slc_end = [
+                self.rslice(shp_end[i], t[0], t[1])
+                if t is not None
+                else tt.arange(shp_end[i])
+                for i, t in enumerate(end)
+            ]
             slc = slc_begin + mid + slc_end
```
Comment (springcoil) [inline pymc3/data.py:357]: In fact this part is the only real change I wanted to make. Why have slc = slc? The changed formatting is by the black code editor.

Comment (twiecki) [reply pymc3/data.py:357]: I think the PR should contain that change only then.

Comment (ferrine) [reply pymc3/data.py:357]: Cc @aseyboldt.
We need to decide if it is OK to gradually reformat code

Comment (springcoil) [reply pymc3/data.py:357]: Agreed. I see no harm in reformatting this with this change.

Comment (fonnesbeck) [reply pymc3/data.py:357]: If we want to apply black formatting it should be done project wide as a separate PR. We can discuss on Friday.

Comment (springcoil) [reply pymc3/data.py:357]: Ok cool. I'll redo the edit

**Pull request merged**